portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -195,7 +196,10 @@ const addSubmissionData = async (dealId, existingApplication) => {
   const updatedPortalActivity = await submissionPortalActivity(existingApplication);
   await addSubmissionDateToIssuedFacilities(dealId);
   await addUkefFacilityIdToFacilities(dealId);
-  await removeChangedToIssued(dealId);
+  // if MIA then handled by central API/tfm API on submission to UKEF
```

Comment:
It's not clear to me what exactly is handled - would be great to be more specific in the comment, something like "MIA activity is..." - or eg "only add X if is AIN" - same for the above comment in code. Just not clear to me, what's meant by it

Reply:
done